gen3utils/gitops/gitops_validator.py

```diff
@@ -417,7 +419,9 @@ def validate_against_dictionary(gitops, data_dictionary):
             "Node: {} in graphql.chartCounts not found in dictionary".format(node),
         )
 
-    for item in graphql.get("homepageChartNodes", []):
+    for item in (
+        gitops.get("components", {}).get("index", {}).get("homepageChartNodes", [])
```

Comment:
Were homepageChartNodes a part of graohql before? Which is now changed to 'components.index.homepageChartNodes'? If so, don't we have to worry about backwards compatibility?

Reply:
no as far as i can tell it was always under components, and [here](https://github.com/uc-cdis/gen3utils/blob/49a2105/gen3utils/gitops/gitops_validator.py#L60-L66) it's fetched from the right place, so i think it was just a mistake we missed here